scripts/build_python.sh

```diff
@@ -206,20 +228,30 @@ if [ -n "$install_virtual_env" ]; then
     fi
 
     source "$ENVIRONMENT_ROOT"/bin/activate
-    "$ENVIRONMENT_ROOT"/bin/python -m ensurepip --upgrade
-    "$ENVIRONMENT_ROOT"/bin/python -m pip install --upgrade "${WHEEL[@]}"
+    python -m ensurepip --upgrade
+fi
 
-    if [ "$install_pytest_requirements" = "yes" ]; then
-        echo_blue "Installing python test dependencies ..."
-        "$ENVIRONMENT_ROOT"/bin/pip install -r "$CHIP_ROOT/scripts/tests/requirements.txt"
-        "$ENVIRONMENT_ROOT"/bin/pip install -r "$CHIP_ROOT/src/python_testing/requirements.txt"
-    fi
+if [ "$install_wheels" = "yes" ]; then
+    # Uninstall the existing wheels and install the new ones to make sure we
+    # have the latest versions even if the wheel version was not bumped.
```

Comment:
Please add a check that the virtualenv should not be the build virtual environment. Maybe we can also support some path somehow.